src/poggit/ci/builder/DefaultProjectBuilder.php

```diff
@@ -164,14 +175,176 @@ protected function build(Phar $phar, RepoZipball $zipball, WebhookProjectModel $
             $localName = $out . substr($file, strlen($in));
             $phar->addFromString($localName, $contents = $reader());
             if(Lang::startsWith($localName, "src/") and Lang::endsWith(strtolower($localName), ".php")) {
-                $this->lintPhpFile($result, $localName, $contents, $localName === $mainClassFile, $project->manifest["lint"] ?? true);
+                $this->lintPhpFile($result, $localName, $contents, $localName === $mainClassFile, $project->manifest["lint"] ?? []);
             }
         }
 
         Virion::processLibs($phar, $zipball, $project, function() use ($mainClass) {
             return implode("\\", array_slice(explode("\\", $mainClass), 0, -1)) . "\\";
         });
 
+        if((($project->manifest["lint"] ?? [])["phpstan"] ?? true)){
```

Comment:
This will crash if people are using the legacy `lint: false`. Consider adding support for `lint: false`.

Reply:
Sorted + Tested.